fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/impl/FHIRPersistenceJDBCImpl.java

```diff
@@ -412,6 +425,10 @@ public <T extends Resource> SingleResourceResult<T> create(FHIRPersistenceContex
                             + ", version=" + resourceDTO.getVersionId());
             }
 
+            if (resourceDTO.getInteractionStatus() == InteractionStatus.MODIFIED) {
+                sendParametersToRemoteIndexService(resourceDTO.getResourceType(), resourceDTO.getLogicalId(), resourceDTO.getId(), 
```

Comment:
I think there's a major issue on this line.  We're passing resourceDTO.getId() into the logicalResourceId parameter but these are two different ids.  `resourceDTO.getId()` is the PK for x_RESOURCES, not x_LOGICAL_RESOURCES.

Downstream (in the fhir-remote-index module) we are also expecting this to be the logicalResourceId (from x_LOGICAL_RESOURCES).  Unfortunately, we don't have that populated in the resourceDTO on insert at the moment.

I think we'll need to decide whether to look it up here or whether to change the message definition (and thus the interface to the remote-index-service) to specify that this is the resourceId (form x_RESOURCES) and not the logicalResourceId.